test/BUILD.gn

```diff
@@ -43,14 +43,14 @@ test("brave_unit_tests") {
 
   configs += [
     "//brave/build/geolocation",
-    "//brave/build/safebrowsing",
   ]
 
   public_deps = [
     "//base",
     "//base/test:test_support",
     "//brave:browser_dependencies",
     "//brave/browser",
+    "//brave/browser/safebrowsing",
```

Comment:
same here with the public dep. If it doesn't have to be a public dep then it shouldn't be because it will cause larger than necessary rebuilds